features/step_definitions/user_steps.rb

```diff
@@ -43,7 +43,7 @@
   end
 end
 
-Given /^I am logged in as "([^\"]*)" with password "([^\"]*)"$/ do |login, password|
+Given /^I am logged in as "([^\"]*)" with password "([^\"]*)"(?:( with preferences set to hidden warnings and no additional tags))?$/ do |login, password, hidden|
```

Comment:
Since the additional tags are hidden like warnings rather and not removed altogether, it should either be "hidden warnings and additional tags" (or "hidden warnings and hidden additional tags" if you want to make it extra clear "hidden" applies to both).